hazelcast/src/main/java/com/hazelcast/config/AbstractXmlConfigHelper.java

```diff
@@ -288,10 +289,7 @@ public static String cleanNodeName(final Node node) {
     }
 
     protected static boolean getBooleanValue(final String value) {
-        String lowerCaseValue = StringUtil.lowerCaseInternal(value);
-        return "true".equals(lowerCaseValue)
-                || "yes".equals(lowerCaseValue)
-                || "on".equals(lowerCaseValue);
+        return parseBoolean(value);
```

Comment:
Actually `equalsIgnoringCase` is black art all on its own, it doesn't rely just on `toLowerCase`. But I changed it back just to remove this worry.